source/delphi_gui/pasdoc_gui.dpr

```diff
@@ -1,4 +1,4 @@
-program pasdoc_gui;
+program pasdoc_gui_Rio;
```

Comment:
Same comment here: do not rename programs that are not only for Rio :) Please revert this change, so program name stays `pasdoc_gui`.

Reply:
Same reason here.